src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -586,8 +587,11 @@ ucs_status_t uct_dc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
     ucs_assert(ep->dci != UCT_DC_MLX5_EP_NO_DCI);
 
     UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
+    sn = txwq->sig_pi;
```

Comment:
minor: imo better put it as "else" in line 620

Reply:
there is jump to ```out``` on line 601 - we have to duplicate evaluation there